src/components/ResultsMap.vue

```diff
@@ -1,10 +1,18 @@
 <template>
 	<section>
-		<results-list-item :result="this.selectedMarker"></results-list-item>
 		<p>Zoom out on map to ensure all result markers are visible.</p>
 		<GmapMap
 			:center="centerMap"
 			:zoom="10"
+			:options="{
+				zoomControl: true,
+				mapTypeControl: false,
+				scaleControl: false,
+				streetViewControl: false,
+				rotateControl: false,
+				fullScreenControl: true,
+				disableDefaultUI: false	
+			}"
 			map-type-id="terrain"
```

Comment:
Thanks for the work on the map appearance. The removal of some of the controls really improves appearance without sacrificing UX.

Reply:
Yes, I agree with Stacy! When you showed us these changes locally, I was impressed by how you were able to sift through which controls were necessary to maintain good UX.